atomic_reactor/rpm_util.py

```diff
@@ -0,0 +1,89 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+image_component_rpm_tags = [
+    'NAME',
+    'VERSION',
+    'RELEASE',
+    'ARCH',
+    'EPOCH',
+    'SIZE',
+    'SIGMD5',
+    'BUILDTIME',
+    'SIGPGP:pgpsig',
+    'SIGGPG:pgpsig',
+]
+
+
+def rpm_qf_args(tags=image_component_rpm_tags, separator=';'):
```

Comment:
Landscape.io complains that tags takes a dangerous default value because it is mutable. We should either default to `None` and handle the real default in the body, or add a comment to stop that warning.

Reply:
Went with the None default, since it will read transparently and a comment would require more thinking on the part of a reader.